app/views/manage_assessments/coverages/new.html.erb

```diff
@@ -1,5 +1,14 @@
 <%= simple_form_for @coverage, url: manage_assessments_course_coverages_path(course_id: @coverage.course.id) do |form| %>
-  <%= form.input :subject_id, collection: Subject.sorted_by_number, label_method: :to_s %>
-  <%= form.input :outcome_id, collection: @coverage.course.outcomes, label_method: :to_short_s %>
+  <%= form.input :subject_id,
+    collection: Subject.sorted_by_number,
+    label_method: :to_s %>
+
+  <%= form.simple_fields_for :outcome_coverages do |fields| %>
+    <%= fields.input :outcome_id,
+      collection: @coverage.course.outcomes,
+      label_method: :to_short_s,
+      required: fields.options[:child_index] == 0 %>
```

Comment:
when you call `fields_for`, the form build object keeps track of which item in the array of associated objects you are on. This sets the `outcome_id` field to be `required: true` when we're rendering the 0th index (or the first item).